testdata/test-catalog.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterCatalog
+metadata:
+  finalizers:
+  - olm.operatorframework.io/delete-server-cache
+  labels:
+    olm.operatorframework.io/metadata.name: test-catalog
```

Comment:
This label is automatically added.

Reply:
I think this file might actually be vestigial, I'm doing another test run but it doesn't appear to be used anywhere. I'll remove it if that's the case (but also thanks, that's good to know!).